resources/lang/de_DE/default.po

```diff
@@ -536,6 +536,9 @@ msgstr "Engel wurde als angekommen markiert."
 msgid "Reset"
 msgstr "Zurücksetzen"
 
+msgid "Reset arrival for %s?"
+msgstr "Ankunft für %s zurücksetzen?"
```

Comment:
Imho its not possible to reset an arrival it's only possible to reset "marked as arrived"
same thing in German